src/deepsparse/pipeline.py

```diff
@@ -26,16 +26,11 @@
 from pydantic import BaseModel, Field
 
 from deepsparse import Context, Engine, MultiModelEngine, Scheduler
+from deepsparse.base_pipeline import BasePipeline
```

Comment:
any way to expose `BasePipeline` from within `pipeline.py`? seems a bit strange from a user perspective to need to include a separate path to get a related class to Pipeline

Reply:
@bfineran

Reply:
@bfineran

Reply:
@mgoin wanted to separate the "base" pipeline with the "engine" pipeline (which is keeping the `Pipeline` name to minimize breaking points) which is why the files are separate.

@dsikka agree that it makes sense from a user perspective to add the `BasePipeline` and registry classes/functions to the `__all__` list in `pipeline.py` so the user imports are a little bit more clear semantically.